client/src/types/global.d.ts

```diff
@@ -0,0 +1,17 @@
+declare const IS_A11Y_MODE: boolean | undefined;
```

Comment:
Looks like we have yet another list of globals to maintain 😞 

vscode's eslint is still acting up on this file, claiming these are all unused variables. I tried adding `*.d.ts` to eslintignore but it's still complaining.